### PR TITLE
[Frontend] [ONNX] Support sequence_lens of GRU

### PR DESCRIPTION
**Pull request by jacquesguan**
Support convert sequence_lens input of GRU.

### CODE REVIEW

**Comment by jacquesguan** [comment]
Could you please review this PR @masahi? Thanks!

**Comment by masahi** [comment]
@tvm-bot rerun

**Comment by masahi** [comment]
cc @vvchernov @AndrewZhaoLuo

**Review by vvchernov — COMMENTED** [review]
LGTM. There are two simple corrections: update GRU_cell description and test params for correct check of new feature

File: python/tvm/relay/frontend/common.py [hunk 1]
```diff
@@ -737,6 +737,7 @@ def gru_cell(
     n_act=_op.tanh,
     backwards=False,
     linear_before_reset=True,
```
[the remark this answers is not in the document]
Comment (jacquesguan) [reply python/tvm/relay/frontend/common.py:739]: Done.

**Review by vvchernov — APPROVED** [review]
LGTM. Hello @masahi, could you recheck and merge this RP?

**Review by masahi — APPROVED** [review]

**Pull request merged**